packages/mobile/src/account/DollarEducation.tsx

```diff
@@ -20,7 +20,7 @@ export class DollarEducation extends React.Component<Props> {
   goToSend = () => {
     this.props.setEducationCompleted()
     CeloAnalytics.track(CustomEventNames.send_dollar_nux)
-    navigate(Screens.Send)
+    navigate(Screens.Send, {})
```

Comment:
I think if params are never needed it can be re typed to accept undefined so you dont need empty object

@cmcewen  is that right?

Reply:
My bad, will add `| undefined` in `navigator/types.tsx`